paket.lock

```diff
@@ -794,19 +799,17 @@ NUGET
       Microsoft.NETCore.Platforms (>= 1.1) - restriction: || (&& (== net472) (< net46)) (== net6.0) (== netcoreapp2.1) (== netstandard2.0)
       Microsoft.NETCore.Targets (>= 1.1) - restriction: || (&& (== net472) (< net46)) (== net6.0) (== netcoreapp2.1) (== netstandard2.0)
       System.Runtime (>= 4.3) - restriction: || (&& (== net472) (< net46)) (== net6.0) (== netcoreapp2.1) (== netstandard2.0)
-    Microsoft.Win32.Registry (5.0) - restriction: || (&& (== net472) (< net45)) (&& (== net472) (< net461)) (&& (== net472) (>= net6.0)) (&& (== net472) (>= netcoreapp3.1)) (&& (== net472) (>= netstandard2.1)) (== net6.0) (== netcoreapp2.1) (== netstandard2.0)
+    Microsoft.Win32.Registry (5.0) - restriction: || (&& (== net472) (< net461)) (&& (== net472) (>= net6.0)) (&& (== net472) (>= netcoreapp3.1)) (== net6.0) (== netcoreapp2.1) (== netstandard2.0)
       System.Buffers (>= 4.5.1) - restriction: || (&& (== net472) (>= monoandroid) (< netstandard1.3)) (&& (== net472) (>= monotouch)) (&& (== net472) (< net46)) (&& (== net472) (>= xamarinios)) (&& (== net472) (>= xamarinmac)) (&& (== net472) (>= xamarintvos)) (&& (== net472) (>= xamarinwatchos)) (&& (== net6.0) (>= monoandroid) (< netstandard1.3)) (&& (== net6.0) (>= monotouch)) (&& (== net6.0) (< netcoreapp2.0)) (&& (== net6.0) (>= xamarinios)) (&& (== net6.0) (>= xamarinmac)) (&& (== net6.0) (>= xamarintvos)) (&& (== net6.0) (>= xamarinwatchos)) (&& (== netcoreapp2.1) (>= monoandroid) (< netstandard1.3)) (&& (== netcoreapp2.1) (>= monotouch)) (&& (== netcoreapp2.1) (< netcoreapp2.0)) (&& (== netcoreapp2.1) (>= xamarinios)) (&& (== netcoreapp2.1) (>= xamarinmac)) (&& (== netcoreapp2.1) (>= xamarintvos)) (&& (== netcoreapp2.1) (>= xamarinwatchos)) (== netstandard2.0)
       System.Memory (>= 4.5.4) - restriction: || (&& (== net472) (< net46)) (&& (== net472) (>= netcoreapp2.0)) (&& (== net472) (>= uap10.1)) (&& (== net6.0) (< netcoreapp2.0)) (&& (== net6.0) (< netcoreapp2.1)) (&& (== net6.0) (>= uap10.1)) (&& (== netcoreapp2.1) (< netcoreapp2.0)) (&& (== netcoreapp2.1) (>= uap10.1)) (== netstandard2.0)
       System.Security.AccessControl (>= 5.0)
       System.Security.Principal.Windows (>= 5.0)
-    Microsoft.Win32.SystemEvents (6.0) - restriction: || (== net6.0) (&& (== netcoreapp2.1) (>= netcoreapp3.1)) (&& (== netstandard2.0) (>= netcoreapp3.1))
+    Microsoft.Win32.SystemEvents (6.0) - restriction: || (&& (== net472) (>= net7.0)) (== net6.0) (&& (== netcoreapp2.1) (>= netcoreapp3.1)) (&& (== netstandard2.0) (>= netcoreapp3.1))
     Mono.Cecil (0.11.4)
     Mono.Posix.NETStandard (1.0)
-    MSBuild.StructuredLogger (2.1.545)
-      Microsoft.Build (>= 16.10)
-      Microsoft.Build.Framework (>= 16.10)
-      Microsoft.Build.Tasks.Core (>= 16.10)
-      Microsoft.Build.Utilities.Core (>= 16.10)
+    MSBuild.StructuredLogger (2.1.820)
```

Comment:
2.1.545 August 2021 to 2.1.820  May 2023. https://www.nuget.org/packages/MSBuild.StructuredLogger

Reply:
Can we make it wildcard? `2.1.*`

Reply:
Looks like it changes pretty often

Reply:
Should be `(>= 2.1.0)` right?